test/relations.test.js

```diff
@@ -1947,30 +2671,68 @@ describe('relations', function () {
           supplierId = supplier.id;
           should.not.exist(e);
           should.exist(supplier);
-          supplier.account.create({accountNo: 'a01'}, function (err, account) {
+          supplier.account.create({accountNo: 'a01', block: false}, function (err, account) {
             supplier.account(function (e, act) {
               accountId = act.id;
               should.not.exist(e);
               should.exist(act);
               act.should.be.an.instanceOf(Account);
+              act.block.should.be.false;
               supplier.account().id.should.equal(act.id);
               act.supplierName.should.equal(supplier.name);
               done();
             });
           });
-        });
+        });
+      });
+    });
+
+    it('should find record that match scope', function (done) {
+      Account.updateAll({ block: true }, function (err) {
+        Supplier.findById(supplierId, function (err, supplier) {
+          supplier.account(function (err, account) {
+            should.not.exists(account);
+            done();
```

Comment:
Hmm, there is no comment here now, have you removed it recently? I am not familiar with these tests, I am ok to keep this particular block of code as it is now, unless @raymondfeng knows a better solution.